packages/framework/tinylicious-client/src/test/TestDataObject.ts

```diff
@@ -16,7 +16,7 @@ export class TestDataObject extends DataObject {
         {},
     );
 
-    public constructor(props: IDataObjectProps) {
```

Comment:
I don't think we should special-case constructor when it comes to access modifiers.

Reply:
Agreed; made a note here: https://github.com/microsoft/FluidFramework/pull/9502/files#r835463433